microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -5,7 +5,7 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
```

Comment:
checking, this might be dup comment, might be done.
To check status, see:
Latest Files Changed view:
https://github.com/MicrosoftDocs/edge-developer/pull/2339/files
Rendered:
https://github.com/MicrosoftDocs/edge-developer/blob/user/v-danwesley/gs-view-page-resources/microsoft-edge/devtools-guide-chromium/resources/index.md

Reply:
done in local copy